test/remote/gateways/remote_stripe_payment_intents_test.rb

```diff
@@ -126,12 +126,15 @@ def test_create_payment_intent_with_return_url
   end
 
   def test_create_payment_intent_with_metadata
+    suffix = 'SUFFIX'
```

Comment:
I like this way of setting a variable instead of using two different strings down below (one when setting, and one when asserting). 👍